ONSdigital/ssdc-rm-support-tool
# 993 follow up work after removing failed job rows

# Motivation and Context
Cancelling a Job clears the JobRow table of bad job rows, however this means there's no data for the error CSVs created by the UI buttons, so the buttons should be removed and a speed bump should be added when cancelling a job

# What has changed
Removed error CSV download buttons on cancelled jobs and added a speed bump popup before the cancel happens

# How to test?
Build & run in docker
Upload valid sample CSV containing errors
Try cancelling the job
Click 'No' and confirm the job remains uncanceled
Try cancelling again
Click 'Yes' and confirm that the job is cancelled and the option to download error CSVs has disappeared
